src/lib/testrail.ts

```diff
@@ -1,15 +1,23 @@
 const axios = require('axios');
-const chalk = require('chalk');
+const find = require('find');
+const request = require('request');
```

Comment:
This PR added `find` and `request` as dependencies, but they aren't listed in `package.json`

Reply:
Nice catch!

When I added those dependencies, it got me thinking... Easiest thing for us to do would be to add new deps, but I think it would be the best if we figure out a way to use `axios` instead of `request` and regular node API instead of `find`. 

If you are willing to, it would be great if you can make PR with solution, if not, we can always include new packages :)